src/api/grainConfig.js

```diff
@@ -49,10 +60,11 @@ export function toDistributionPolicy(x: GrainConfig): DistributionPolicy {
       `balanced budget must be nonnegative integer, got ${balancedPerWeek}`
     );
   }
-  const allocationPolicies = [];
+
+  const allocationPoliciesDeprecated: Array<AllocationPolicy> = [];
```

Comment:
Why not just push them directly into the allocationPolicies array?

Reply:
Ack, but I like thinking of these processes as a tree.  One branch handles `x.allocationPolicies` and the other handles the deprecated policies.